test/e2e/tests/metrics/errors.spec.js

```diff
@@ -793,30 +794,6 @@ describe('Sentry errors', function () {
     });
   });
 
-  it('should have no policy gaps for UI controller state @no-mmi', async function () {
-    await withFixtures(
-      {
-        fixtures: new FixtureBuilder().build(),
-        ganacheOptions,
-        title: this.test.fullTitle(),
-      },
-      async ({ driver }) => {
-        await driver.navigate();
-        await driver.findElement('#password');
-
-        const fullUiState = await driver.executeScript(() =>
-          window.stateHooks?.getCleanAppState?.(),
-        );
-
-        const missingState = getMissingProperties(
-          fullUiState.metamask,
-          SENTRY_UI_STATE.metamask,
-        );
-        assert.deepEqual(missingState, {});
-      },
-    );
-  });
-
```

Comment:
I removed this test as its wrong and contradictory to test just below it. With the addition of `lastInteractedConfirmationInfo` this flaw showed up.

Reply:
Nice 👍🏼